src/track/trackmetadatataglib.cpp

```diff
@@ -2217,9 +2220,12 @@ bool exportTrackMetadataIntoID3v2Tag(TagLib::ID3v2::Tag* pTag,
     // or if the track has a Work field then store the Grouping in a
     // GRP1 frame instead of using TIT1.
     // See also: importTrackMetadataFromID3v2Tag()
-    if (trackMetadata.getTrackInfo().getWork().isNull() &&
-            pTag->frameListMap().contains("GRP1")) {
-        // Stick to traditional mapping if the new GRP1
+    if (
+#if defined(__EXTRA_METADATA__)
+            trackMetadata.getTrackInfo().getWork().isNull() &&
```

Comment:
Is this condition true? 
I wonder if iTunes use the work tag without GRP1 tag for mp4 files. 
What do others, if we consider the iTunes way as a hotfix for TIT1 missuse?

Reply:
The corresponding (custom) MP4 atoms are well defined without any context-sensitive switching.

Maybe we should use the new mapping if *work* is present **or** *movement* is present **or** GRP1 is present?

Reply:
Done. The new version of the export code now clearly distinguishes between either TIT1 or GRP1/TIT1/MVNM.